tests/test_key.py

```diff
@@ -120,6 +126,120 @@ def test_sslib_equality(self):
             self.assertEqual(sslib_key_2, sslib_key)
 
 
+@unittest.skipIf(not HAVE_GPG, "gpg not found")
+class TestGPGKey(unittest.TestCase):
```

Comment:
Let's combine this test case with `test_signer.TestGPGRSA` to save some redundant test code in test_key, test_signer and test_gpg.

Reply:
you mean using `GPGKey` in `test_signer.TestGPGRSA` or something else?

Reply:
Yep, or keep your newly added test case and remove `test_signer.TestGPGRSA`. Either way is fine.